lnd_grpc/lnd_grpc.py

```diff
@@ -483,15 +485,28 @@ def fee_report(self):
         response = self.lightning_stub.FeeReport(request)
         return response
 
-    def update_channel_policy(self, **kwargs):
-        if 'chan_point' in kwargs:
-            funding_txid, output_index = kwargs.get('chan_point').split(':')
-            _channel_point = self.channel_point_generator(funding_txid=funding_txid,
-                                                          output_index=output_index)
-            kwargs['chan_point'] = _channel_point
-        if 'global' not in kwargs:
-            kwargs['global'] = 1
-        request = ln.PolicyUpdateRequest(**kwargs)
+    def update_channel_policy(self,
+                              chan_point: str,
+                              is_global: bool = False,
+                              base_fee_msat: int = 1000,
+                              fee_rate: float = 0.000001,
+                              time_lock_delta: int = 144
+                              ):
+        if chan_point:
+            funding_txid, output_index = chan_point.split(':')
+            channel_point = self.channel_point_generator(funding_txid=funding_txid,
+                                                         output_index=output_index)
+        else:
+            channel_point = None
+
+        request = ln.PolicyUpdateRequest(
+            chan_point=channel_point,
+            base_fee_msat=base_fee_msat,
+            fee_rate=fee_rate,
+            time_lock_delta=time_lock_delta
+        )
+        if is_global:
```

Comment:
@PierreRochard I wonder here if we should choose to mimic the cli flow which, if no chan_point is given, will set global automagically?

Could be something like: 

```
if is_global or channel_point == None:
    ...
```